packages/backend-function/API.md

```diff
@@ -6,14 +6,35 @@
 
 import { S3Client } from '@aws-sdk/client-s3';
 
+declare namespace __export__get_amplify_clients_configuration {
+    export {
+        DataClientEnv,
+        getResourceConfig,
+        getLibraryOptions,
+        InvalidConfig,
+        DataClientError,
+        DataClientConfig,
+        DataClientReturn,
+        getAmplifyDataClientConfig
+    }
+}
+export { __export__get_amplify_clients_configuration }
+
+declare namespace __export__runtime {
+    export {
+        getAmplifyDataClientConfig

```

Comment:
It's probably redundant to export this from the index path where it is exported from. Should I remove this, or is it good to keep it since this is where it is consumed?

Reply:
We should only have `index`.

I.e. match this entry point from package.json.

```
    "./runtime": {
      "types": "./lib/runtime/index.d.ts",
      "import": "./lib/runtime/index.js",
      "require": "./lib/runtime/index.js"
```

Reply:
the goal is to have api extractor "see" that entry point and report types accessible from there.

Reply:
Updated.